runtime/core/decoder/torch_asr_decoder.h

```diff
@@ -140,6 +141,19 @@ class TorchAsrDecoder {
   std::vector<std::vector<float>> cached_feature_;
   bool start_ = false;
 
+  // NOTE(xcsong): In order to simplify the comparison of different APIs,
+  //  all the data required by the two APIs are retained.
+
+  // API_v2
+  // cnn_cache_: (elayers, batch, cnn_kernel - 1, hidden_dim)
+  torch::Tensor cnn_cache_ = torch::zeros({12, 0, 0, 0});
```

Comment:
12 is hard code here?

Reply:
Any way to simplify it? can we use torch::zeros(0) to denote that is an empty cache?